Resfeber/CoreData/CoreDataStack.swift

```diff
@@ -18,8 +18,7 @@ class CoreDataStack {
         return NSManagedObjectModel(contentsOf: modelURL)!
     }()
     
-    private init() {
-    }
+    public init() {}
```

Comment:
You want to leave this as a private init() because you should only be using the `shared` instance of the `CoreDataStack`, and making this private will prevent developer and yourself from accidentally creating multiple core data stacks

Reply:
Same reason as the `open` access level modifier added above. This issue is also resolved by adding `@testable` in the unit test files, so I will change this back to `private`
```suggestion
    private init() {}
```